### ISSUE TITLE
div id selectors not working

### ISSUE THREAD

**Issue by raindropsonsnowdrops** [microsoft/vscode#137294]
We have written the needed
Issue Type: <b>Bug</b>

Hi,
I am having trouble with the div id selectors. Here is an example of what I'm doing:

html:
```
<div id="one">Lorem ipsum...</div>
```

and CSS:
```
#one {background-color: blue;}
```

The changes are not showing up on my browser. Is there another sign I should use, (other than the # or .) or is there a bug? Maybe something changed in the update that I am not aware of?

Thanks.

VS Code version: Code 1.62.2 (3a6960b964327f0e3882ce18fcebd07ed191b316, 2021-11-11T20:56:38.428Z)
OS version: Windows_NT x64 10.0.19041
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i3-3225 CPU @ 3.30GHz (4 x 3293)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: unavailable_off<br>opengl: enabled_on<br>rasterization: unavailable_off<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|3.97GB (0.68GB free)|
|Process Argv|--crash-reporter-id 99ab01ad-cd8c-412b-bc3b-91ee50994803|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
debugger-for-edge|msj|1.0.15

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonvspyt551cf:30345471
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30384386
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis200cf:30386380
vsaa593:30376534
vssld246:30386377

```

</details>

 data into your clipboard because it was too large to send. Please paste.

**Comment by aeschli** [comment]
I don't see how this is related to VS Code. Sounds more like a general HTML question.

Can you ask the question in an HTML forum, e.g. on stackoverflow ?

**Comment by raindropsonsnowdrops** [comment]
Hi

Other people have used the **_exact same code_** and it worked for them.

Everything else I've done has worked. I've tried uninstalling and installing again, it's just not working.

I don't think there is a problem with my code unless there has been an update, which I don't think there is.

I have tried viewing in many different browsers, none of which worked.

I can't see a problem with my code and have worked on this for days. 

I think the problem must be with VS Code.

Thanks for your reply.
